### PR TITLE
Lk pd 2259 add dropseq

### PR DESCRIPTION
**Pull request by ekiernan**
### Description

Adding Drop-seq metrics to Optimus. This will create a new output metrics file for Picard cell metrics.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [x] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [x] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by aawdeh — APPROVED** [review]
LGTM!

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-# 5.9.0 
+
```
Comment (jessicaway) [inline pipelines/skylab/optimus/Optimus.changelog.md:1]: Was the version number removed here? I looks like maybe we mean for these bullet points to be added to the 5.8.3 entry?

File: pipelines/skylab/multiome/Multiome.wdl [hunk 2]
```diff
@@ -6,6 +6,7 @@ import "../../../pipelines/skylab/optimus/Optimus.wdl" as optimus
 workflow Multiome {
     String pipeline_version = "1.0.2"
 
+
```
Comment (jessicaway) [inline pipelines/skylab/multiome/Multiome.wdl:9]: do we normally include 2 new lines?

**Comment by jessicaway** [comment]
Not sure why pullapprove is requesting George's approval. Since this does not touch anything used in Arrays, I have added the eng-only label to bypass that requirement.

File: pipelines/skylab/optimus/Optimus.changelog.md (hunk 1 above)
Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.changelog.md:1]: I moved the bullets to other changelog entry

**Pull request merged**